File: MobileNaim/Pidev3/src/com/mycompany/myapp/entities/ratingactivite.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package com.mycompany.myapp.entities;

/**
 *
 * @author ASUS
 */
public class ratingactivite {
    private int IdRate;
    private String Intitule;
    private int Rate;

    public ratingactivite() {
    }

    public ratingactivite(int IdRate, String Intitule, int Rate) {
        this.IdRate = IdRate;
        this.Intitule = Intitule;
        this.Rate = Rate;
    }
    
     public ratingactivite( String Intitule, int Rate) {
      
        this.Intitule = Intitule;
        this.Rate = Rate;
    }

    public int getIdRate() {
        return IdRate;
    }

    public void setIdRate(int IdRate) {
        this.IdRate = IdRate;
    }

    public String getIntitule() {
        return Intitule;
    }

    public void setIntitule(String Intitule) {
        this.Intitule = Intitule;
    }

    public int getRate() {
        return Rate;
    }

    public void setRate(int Rate) {
        this.Rate = Rate;
    }

    @Override
    public String toString() {
        return "ratingactivite{" + "IdRate=" + IdRate + ", Intitule=" + Intitule + ", Rate=" + Rate + '}';
    }
     
    
}

File: web zied/PiDevF2/PiDev/src/PidevBundle/Entity/Ratingactivite.php
<?php

namespace PidevBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Ratingactivite
 *
 * @ORM\Table(name="ratingactivite")
 * @ORM\Entity
 */
class Ratingactivite
{
    /**
     * @var integer
     *
     * @ORM\Column(name="IdRate", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idrate;

    /**
     * @var string
     *
     * @ORM\Column(name="Intitule", type="string", length=25, nullable=false)
     */
    private $intitule;

    /**
     * @var integer
     *
     * @ORM\Column(name="Rate", type="integer", nullable=false)
     */
    private $rate;

    /**
     * @return int
     */
    public function getIdrate()
    {
        return $this->idrate;
    }

    /**
     * @param int $idrate
     */
    public function setIdrate($idrate)
    {
        $this->idrate = $idrate;
    }

    /**
     * @return string
     */
    public function getIntitule()
    {
        return $this->intitule;
    }

    /**
     * @param string $intitule
     */
    public function setIntitule($intitule)
    {
        $this->intitule = $intitule;
    }

    /**
     * @return int
     */
    public function getRate()
    {
        return $this->rate;
    }

    /**
     * @param int $rate
     */
    public function setRate($rate)
    {
        $this->rate = $rate;
    }


}

